apps/src/codebridge/Preview/Preview.tsx

```diff
@@ -0,0 +1,30 @@
+import {useCodebridgeContext} from '@codebridge/codebridgeContext';
+import ControlButtons from '@codebridge/Console/ControlButtons';
+import React from 'react';
+
+import codebridgeI18n from '@cdo/apps/codebridge/locale';
+import PanelContainer from '@cdo/apps/lab2/views/components/PanelContainer';
+
+import moduleStyles from './preview.module.scss';
+
+const Preview: React.FunctionComponent = () => {
```

Comment:
I wonder if we could tweak name to help reader understand what this component is used for. Maybe `VisualizationPreview` (similar to `VisualizationColumn`?) But that's pretty wordy. Don't have a strong feeling here.

Reply:
The panel itself is titled `Preview`, which is why I went with this, but I also don't love it (especially because of the overlap with the html preview panel). I think `VisualizationPreview` is a bit redundant. I think `Visualization` maybe makes more sense than `Preview`, but since the panel is called `Preview` it might be easier to keep the name consistent for folks to find the file.

Alternatively we could call this `MiniAppPreview`, although we could theoretically put something besides a mini app in here 🤔 I'm open to suggestions! Things may also become clearer as we do more development on this, the names aren't set in stone.

Reply:
Ooo...I like `MiniAppPreview`, but I agree to wait until we do more work in this area. Thanks!

Reply:
I'll rename to `MiniAppPreview`, we can always rename it in the future if we end up doing things other than mini apps here. Thanks!

Reply:
With this name update, I think `FilePreview` makes more sense now. https://github.com/code-dot-org/code-dot-org/pull/62978#discussion_r1888767392 - thanks!

Reply:
done!